ios/RNSScreen.m

```diff
@@ -505,6 +529,81 @@ - (void)notifyFinishTransitioning
   _previousFirstResponder = nil;
 }
 
++ (UIInterfaceOrientationMask)interfaceOrientationMaskForStackOrientationMask:(RNSScreenOrientationMask)stackOrientationMask
```

Comment:
One better place where I think this could be placed is RCTConvert interface. You could add RCTCOnvert extension for UIInterfaceOrientationMask, this way you no longer need to have to keep you own type and handle conversion from JS param to your own enum and then again from your enum to UIInterfaceOrientationMask

You can check here for an example usage: https://github.com/facebook/react-native/blob/master/React/Views/RCTConvert%2BCoreLocation.m

In that example there is a conversion defined for CLLocationCoordinate2D type. This way, in view managers and in native modules it is allowed to use CLLocationCoordinate2D type as an argument, framework will handle conversion automatically when that type is used as an argument.

Reply:
It is definitely a good idea.